<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qqtdicttest8</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="text">
       <string>UTF-8</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string>,</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="text">
       <string>&quot;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="lineEdit_11">
      <property name="text">
       <string>,</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QTextBrowser" name="textBrowser_2"/>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Translate</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="lineEdit_21">
      <property name="text">
       <string>&quot;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="lineEdit_31">
      <property name="text">
       <string>UTF-8</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>612</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
